tests/src/test/java/ca/bc/gov/open/jagefilingapi/qa/cucumber/Options/RunCucumberTest.java

```diff
@@ -0,0 +1,24 @@
+package ca.bc.gov.open.jagefilingapi.qa.cucumber.Options;
+
+import io.cucumber.junit.Cucumber;
+import io.cucumber.junit.CucumberOptions;
+import org.junit.runner.RunWith;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(
+         features = {"src/test/java/features"},
+         glue ={"stepDefinitions"},
+        // tags = "@backend",
```

Comment:
if commented out, do we need this? or we can mark for a TODO?

Reply:
Fixed. Thanks!